WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.kt

```diff
@@ -208,6 +208,11 @@ class MeFragment : Fragment(R.layout.me_fragment), OnScrollToTopListener {
             rowDebugSettings.setOnClickListener {
                 requireContext().startActivity(Intent(requireContext(), DebugSettingsActivity::class.java))
             }
+            rowDesignSystem.isVisible = true
+            designSystemDivider.isVisible = true
+            rowDesignSystem.setOnClickListener {
+                ActivityLauncher.viewDesignSystemForResult(activity)
```

Comment:
This could be just like L:209 `requireContext().startActivity(Intent(requireContext(), DesignSystemActivity::class.java))`.   It is not necessary to use `ActivityLauncher`